src/utils/config.ts

```diff
@@ -0,0 +1,43 @@
+import { createConfig, http, cookieStorage, createStorage } from 'wagmi';
+import { localhost, sepolia } from 'wagmi/chains';
+import { rainbowWallet, walletConnectWallet, injectedWallet } from '@rainbow-me/rainbowkit/wallets';
+import { connectorsForWallets } from '@rainbow-me/rainbowkit';
+
+import { getConfig } from '../config';
+
+const { PROJECT_ID } = getConfig();
+
+const getWallets = () => {
+  if (PROJECT_ID) {
+    return [injectedWallet, rainbowWallet, walletConnectWallet];
+  } else {
+    return [injectedWallet];
+  }
+};
+
+const connectors = connectorsForWallets(
+  [
+    {
+      groupName: 'Recommended',
+      wallets: getWallets(),
+    },
+  ],
+  {
+    appName: 'Web3 React boilerplate',
```

Comment:
What is this particulary? Do we need to change it in case we want to build something with the boilerplate? If so, it should be an ENV variable

Reply:
Agreed, let's do it in another PR

Reply:
It is a required value for Coinbase Wallet connector (https://wagmi.sh/react/api/connectors/coinbaseWallet#appname), but doesnt need to be an environment variable bc this default value does not affect any interactions.